tests/unit/EffectiveDateTime.spec.ts

```diff
@@ -122,10 +122,9 @@ describe('Effective Date Time component', () => {
     const radioIsFutureEffective = radioInput.at(1)
 
     await radioIsFutureEffective.trigger('click')
-    wrapper.vm.dateText = new Date().toISOString().split('T')[0]
```

Comment:
Since you've removed this, is there another test in this file that verified a correct date (or no input, or whatever this previously tested)?

Reply:
There was another place that set dateText but didn't make any difference for the test validity. I removed it as well.

Reply:
Does this component now need a new test to verify a valid date?

Reply:
It has other tests that test for the valid date.

Reply:
ok